charts/bctlquickstart/values.yaml

```diff
@@ -74,4 +74,4 @@ tolerations: []
 
 affinity: {}
 
-serviceUrl: "cloud.bastionzero.com"
+serviceUrl: "https://cloud.bastionzero.com"
```

Comment:
I don't think this is going to work for out agent, our agent expects cloud.bastionzero.com to be set for the service url. See here:
<img width="705" alt="2021-11-23 at 10 13 PM" src="https://user-images.githubusercontent.com/28202162/143166313-aa51ebe6-230a-4ea4-a5e9-a2746274393d.png">


That is why I added this: https://github.com/bastionzero/charts/pull/5/files#diff-197d38e15636244b15d85d39322c397d22c6a8b3fa07aade2e7bbcaf94261941L37

Reply:
Not following what the issue is exactly. I removed `https://` prefix from https://github.com/bastionzero/charts/pull/5/files#diff-197d38e15636244b15d85d39322c397d22c6a8b3fa07aade2e7bbcaf94261941L37 and instead the full url is expected to be passed in as the helm variable. Shouldnt this change be transparent unless you are explicitly overriding the service url helm variable?

Reply:
You removed it from the param you pass to the _quickstart_ container, but see here: https://github.com/bastionzero/charts/blob/feat/system-test/charts/bctlquickstart/templates/agent/bctl-agent-deployment.yaml#L36 where that value will now contain a `https://` where it is not expecting one.

Reply:
2a80578 - Removes default env values in the agent deployment yaml

Agent deployment yaml doesnt actually need default values defined for env variables because we will dynamically update these values in the bctlquickstart job https://github.com/bastionzero/charts/blob/82f16f09f6c476ddf24d16dd39035b5497a0c7a9/charts/bctlquickstart/bctl-quickstart/bctl_quickstart/bctl_quickstart/main.py#L29